src/components/List/index.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { ComponentType, memo, ReactNode } from "react";
+import React, { ComponentType, memo, ReactNode, ElementType } from "react";
 import ListRows, { Props as ListRowsProps } from "../ListRows";
 import Empty, { Props as EmptyProps } from "../Empty";
 
 export interface Props<T, U = T, V extends EmptyProps = EmptyProps>
   extends ListRowsProps<T, U> {
   className?: string;
+  Container?: ComponentType<{ className?: string; children: ReactNode }> | ElementType;
```

Comment:
I think `ElementType` just cause type checking ignoring `ComponentType<{ className?: string; children: ReactNode }>`. can't we cast `div` to solve the problem?

Reply:
You are right about type checking issue, I fixed this prop type. Now it's possible to set all HTML tags + valid react components that have required props.
Are you Ok with this idea?